bigframes/dataframe.py

```diff
@@ -797,11 +798,12 @@ def _apply_dataframe_binop(
     def eq(self, other: typing.Any, axis: str | int = "columns") -> DataFrame:
         return self._apply_binop(other, ops.eq_op, axis=axis)
 
+    def __eq__(self, other) -> DataFrame:
+        return self.eq(other)
+
```

Comment:
how about adding `inspect.getdoc(vendored_pandas_frame.DataFrame.__eq__)` to generate the docs?

Reply:
__eq__ = eq

Reply:
use this pattern at bunch of places, however for `__eq__` I created its own docstring with code samples.

Reply:
Moved all dunder docstrings to third_party for compliance reasons (even slight similarity with an existing pandas documentation may be considered as derived) and used this pattern for explicit override.